ballerina/Package.md

```diff
@@ -1,14 +1,60 @@
 ## Overview
-
-[//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+Gemini, developed by Google DeepMind in collaboration with Google Research, is the most advanced and versatile AI model they’ve created. Designed to be multimodal, Gemini [https://blog.google/technology/ai/google-gemini-ai/#sundar-note] seamlessly understands and integrates various types of information, such as text, code, audio, images, and video. This marks a significant step toward building intuitive, expert AI assistants that enhance human capabilities.
 
 ## Setup guide
+To use the Gemini connector, you must have a Google account with API access. So in order to access the APIs you must have a api-key [https://ai.google.dev/gemini-api/docs/api-key]
+
+### Step 1: Create an API Token
+
+1. Naviaget to the get api key [https://ai.google.dev/gemini-api/docs/api-key]
+![get-a-gemini-api-key.png](..%2Fdocs%2Fsetup%2Fresource%2Fget-a-gemini-api-key.png)
```

Comment:
Shall we add images paths as [here](https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/ballerina/Package.md?plain=1#L17) ?

Reply:
Check for other places as well